test/tools/yulInterpreter/Interpreter.h

```diff
@@ -96,6 +96,12 @@ struct InterpreterState
 	void dumpTraceAndState(std::ostream& _out) const;
 };
 
+struct Scope
```

Comment:
It is a rather generic name, is it a big amount of work to move it inside the interpreter class?